app/javascript/react/containers/ReviewsContainer.js

```diff
@@ -27,21 +44,21 @@ class ReviewsContainer extends Component {
     })
     .catch(error => console.error(`Error in fetch: ${error.message}`));
   }
-
   render() {
-
```

Comment:
wait no, you removed MORE whitespace!? haha My poem was about _not_ removing it. haha there should be whitespace between code blocks and method declarations for good style :P